scripts/install_python_prod_dependencies.py

```diff
@@ -140,7 +140,7 @@ def _get_requirements_file_contents() -> Dict[str, str]:
         common.COMPILED_REQUIREMENTS_FILE_PATH, 'r') as f:
         trimmed_lines = (line.strip() for line in f.readlines())
         for line_num, line in enumerate(trimmed_lines, start=1):
-            if not line or line.startswith('#'):
+            if not line or line.startswith('#') or line.startswith('--'):
```

Comment:
```suggestion
            if not line or line.startswith('#') or line.startswith('--hash='):
```

Just to make it clearer what's being excluded with the second `line.startswith`

Reply:
we are skipping the via dependencies that are started with '#', and skipping the hashes that starts with '--hash'.
so changing the condition to `--hash=`